extensions/linz_dataset/README.md

```diff
@@ -0,0 +1,50 @@
+# LINZ Dataset Extension Specification
+
+- **Title:** LINZ Dataset
+- **Identifier:** <https://github.com/stac-extensions/processing/blob/v0.0.1/extensions/linz_dataset/json-schema/schema.json>
+- **Field Name Prefix:** linz_dataset
```

Comment:
Could we split the field definitions into several reusable extensions, leaving this extension to only enforce the requiredness? That way we can evolve the standard with the community, and ensure that the extensions are usable outside of Toitū Te Whenua Land Information New Zealand. For example, the status and change log fields seem like they would fit into the existing [version extension](https://github.com/stac-extensions/version), and it's not clear what item count provides beyond counting the number of item metadata files related to this file.

Reply:
Yes, please do. I did this when there were deadlines for completion of the metadata review, but didn't get it finished before that piece of work wound up. It was clear that more thought would be needed on how these fields interact with the existing extensions (there's likely to be a number of proposals to be made to enhance or extend existing extensions).

STAC was also going through preparation for v1.0.0 release at the time (including splitting extensions out into their own repo) so it was a bit awkward to attempt to do this then.